src/tests/user-profile.ts

```diff
@@ -4,15 +4,15 @@ import { startAsUser, buildUrl, TEST_USERS, submitResults } from '../helpers';
 import { NightwatchBrowser } from 'nightwatch';
 
 const UserProfileTest: BaseTest = {
-  '@disabled': false,
+  '@disabled': true,
```

Comment:
currently completely broken because Chrome doesn't like login from an automated browser. will address here: https://github.com/icgc-argo/e2e-test/issues/44